charts/sonarqube/values.yaml

```diff
@@ -362,14 +362,15 @@ annotations: {}
 
 ## We usually don't make specific ressource recommandations, as they are heavily dependend on
 ## The usage of SonarQube and the surrounding infrastructure.
-## Adjust these values to your needs, but make sure that the memory limit is never under 4 GB
+## Adjust these values to your needs, but make sure that the memory limit is never under 6 GB
+## Those default are based on the default Web -Xmx1G -Xms128m and CE -Xmx2G -Xms128m and Search -Xmx2G -Xms2G settings of SQ sub processes
 resources:
   limits:
     cpu: 800m
-    memory: 4Gi
+    memory: 6144M
   requests:
     cpu: 400m
-    memory: 2Gi
+    memory: 2048M
```

Comment:
The combined minimum heap size should be more that 2Gi...shall we set it to be 2560M (~2,5Gi)?

Reply:
Here its a bit tricky as we need to comply with 3 different editions.

I would have prefered to setup 6G as request and limits, but this would probably break some community editions and this would be too big.

I am open to debate actually, i don't think we have a silver bullet for this.